### DISCUSSION TITLE
How to insert image

### DISCUSSION THREAD

**Discussion by JorgenFP** [ObaraEmmanuel/Formation#35] [Q&A]
Hi, how do I insert an image into my GUI?
Thanks, great app.

**Comment by ObaraEmmanuel** [comment]
Thanks for using Formation studio. Here are the steps you can follow to add an image in your GUI

### Step 1
Place a widget that supports image option like `tk.Label`, `tk.Button`, etc. You can drag them from the Components pane onto the design surface.

![image](https://github.com/user-attachments/assets/73cf2660-9f59-4b9f-bd4c-599f6564da55)

### Step 2
Find the `image` option in the StylePane and click on the "..." to open the file picker

![image](https://github.com/user-attachments/assets/227c4071-a234-4917-a109-50a063835da5)

### Step 3
Select the desired image image from your system.

![image](https://github.com/user-attachments/assets/a06e9163-49a5-4cb1-92fa-99e2c5e12a35)


> Note:
> You can choose whether you want to use absolute or relative paths in `File > settings > Designer > Image options`
